lua/ulx/modules/sh/cfc_arrest.lua

```diff
@@ -0,0 +1,51 @@
+CFCUlxCommands.arrest = CFCUlxCommands.arrest or {}
+local cmd = CFCUlxCommands.arrest
+
+CATEGORY_NAME = "Cleanup"
+
+function cmd.arrest( callingPlayer, targetPlayers )
+    local entities = ents.GetAll()
+    local entCount = 0
+    local entCounts = {}
+    for _, ply in ipairs( targetPlayers ) do
+        entCounts[ply] = 0
+    end
```

Comment:
You can get rid of this loop by defaulting them to 0 below (see other suggestion)